pkgs/applications/misc/djvulibre/default.nix

```diff
@@ -13,10 +13,10 @@ stdenv.mkDerivation rec {
   buildInputs = [ libjpeg libtiff librsvg libiconv ];
 
   meta = with stdenv.lib; {
-    description = "A library and viewer for the DJVU file format for scanned images";
+    description = "The big set of CLI tools to make/modify/optimize/show/export DJVU files";
     homepage = http://djvu.sourceforge.net;
```

Comment:
Quote the url.

Reply:
Oh, thanks, you saying that to me for 2 or 3 time. So..., I am writing write recommendation to myself.

Reply:
Done.